build/test-scripts/integration/common/setup-vm-agent.sh

```diff
@@ -5,4 +5,4 @@
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 source $SCRIPT_DIR/dotnet-install.sh
-install_dotnet 6.0
+install_dotnet --runtime 6.0 --sdk 8.0
```

Comment:
Why does the runtime differ from the sdk? oh is it 8  compiling for 6? Or is runtime 6.0 just needed for tests? Whatever the reason some helpful doco will help the next poor developer

Reply:
I'll add a comment. Basically this is saying install the 6.0 runtime and the 8.0 sdk. There is some hyper simplistic argument parsing going on.